api/migrations/seed/index.js

```diff
@@ -150,6 +150,8 @@ messages.map(message => {
 debug('Connecting to db...');
 // $FlowFixMe
 const db = require('rethinkdbdash')({
+  host: 'rethinkdb',
```

Comment:
This should probably read from `process.env.AWS_RETHINKDB_HOST` and then fallback to `localhost`.